src/client/src/components/DraggableSidebarItem/index.tsx

```diff
@@ -139,12 +141,13 @@ const DraggableSidebarItem = ({
             </div>
           )}
         </div>
-        <CloseSVG
+        {(totalPageCount !== undefined ? totalPageCount > 1 : true) && <CloseSVG
```

Comment:
NIT: Style - I think it looks better (and is consistent with other tags) to start HTML tag in new line.
BUT: I believe that code formatting is not developer's job - it's computer's job. @SahilTara, Did you guys use an autoformatting tool? I remember hearing the word "Prettier" in one of onboarding videos :)
If the answer is yes, then everyone should configure "format on save" and don't think about style any more :)

Reply:
Yeah we used prettier a lot! I too recommend everyone gets prettier. Turn format on save on or else there will be giant diffs between people who have format on save on and those who don't (we experienced such). https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
If you're going to change the formatting of code, it's beneficial to make the reformatting changes as a separate commit so changes in functionality are clear and separate.
Also it's important that everyone is using the same layout/formatting/linting rules as everyone else. Having consistency across the code base makes reviews simpler and helps when maintaining code.

If it is decided that a project should use a particular tool then then should be specified as a recommended extension in the workspace. (see https://code.visualstudio.com/updates/v1_6#_workspace-extension-recommendations)
Any configuration files for extensions should also be checked into the repo so that everyone uses the same tools with the same settings.

Reply:
fixed as suggested. The formatting change is in another commit